src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -30,10 +30,24 @@
  * modules should define a single function that returns an object of all
  * exported functions.
  */
-function RemoteFunctions(experimental) {
+function RemoteFunctions(config) {
     "use strict";
 
+    var experimental = config.experimental;
     var lastKeepAliveTime = Date.now();
+	var req, timeout;
```

Comment:
This commit has `tabs`, while we prefer/enforce 4 spaces. You can easily change them with the `brackets-eslint` extension by right clicking the editor and selecting `Autofix with ESLint`